components/test_raftstore/src/common-test.toml

```diff
@@ -110,5 +110,7 @@ num-threads = 1
 wait-for-lock-timeout = "3s"
 wake-up-delay-duration = "100ms"
 
-[cdc]
```

Comment:
Does this change break compatibility？

Reply:
No, this config file is used for test, and the `cdc` config is only used for stale read test